codepoints.json

```diff
@@ -0,0 +1,243 @@
+    "wordpress": 61728,
+    "wordpress-alt": 62244,
+    "pressthis": 61783,
+    "update": 62563,
+    "screenoptions": 61824,
+    "cart": 61812,
+    "feedback": 61813,
+    "cloud": 61814,
+    "translation": 62246,
+    "tag": 62243,
+    "category": 62232,
+    "archive": 62592,
+    "tagcloud": 62585,
+    "text": 62584,
+    "media-archive": 62721,
+    "media-audio": 62720,
+    "media-code": 62617,
+    "media-default": 62616,
+    "media-document": 62615,
+    "media-interactive": 62614,
+    "media-spreadsheet": 62613,
+    "media-text": 62609,
+    "media-video": 62608,
+    "playlist-audio": 62610,
+    "playlist-video": 62611,
+    "controls-play": 62754,
+    "controls-pause": 62755,
+    "controls-forward": 62745,
+    "controls-skipforward": 62743,
+    "controls-back": 62744,
+    "controls-skipback": 62742,
+    "controls-repeat": 62741,
+    "controls-volumeon": 62753,
+    "controls-volumeoff": 62752,
+    "yes": 61767,
+    "no": 61784,
+    "no-alt": 62261,
+    "plus": 61746,
+    "plus-alt": 62722,
+    "plus-alt2": 62787,
+    "minus": 62560,
+    "dismiss": 61779,
+    "marker": 61785,
+    "star-filled": 61781,
+    "star-half": 62553,
+    "star-empty": 61780,
+    "flag": 61991,
+    "info": 62280,
+    "warning": 62772,
+    "share": 62007,
+    "share1": 62007,
+    "share-alt": 62016,
+    "share-alt2": 62018,
+    "twitter": 62209,
+    "rss": 62211,
+    "email": 62565,
+    "email-alt": 62566,
+    "facebook": 62212,
+    "facebook-alt": 62213,
+    "networking": 62245,
+    "googleplus": 62562,
+    "location": 62000,
+    "location-alt": 62001,
+    "camera": 62214,
+    "images-alt": 62002,
+    "images-alt2": 62003,
+    "video-alt": 62004,
+    "video-alt2": 62005,
+    "video-alt3": 62006,
+    "vault": 61816,
+    "shield": 62258,
+    "shield-alt": 62260,
+    "sos": 62568,
+    "search": 61817,
+    "slides": 61825,
+    "analytics": 61827,
+    "chart-pie": 61828,
+    "chart-bar": 61829,
+    "chart-line": 62008,
+    "chart-area": 62009,
+    "groups": 62215,
+    "businessman": 62264,
+    "id": 62262,
+    "id-alt": 62263,
+    "products": 62226,
+    "awards": 62227,
+    "forms": 62228,
+    "testimonial": 62579,
+    "portfolio": 62242,
+    "book": 62256,
+    "book-alt": 62257,
+    "download": 62230,
+    "upload": 62231,
+    "backup": 62241,
+    "clock": 62569,
+    "lightbulb": 62265,
+    "microphone": 62594,
+    "desktop": 62578,
+    "laptop": 62791,
+    "tablet": 62577,
+    "smartphone": 62576,
+    "phone": 62757,
+    "smiley": 62248,
+    "index-card": 62736,
+    "carrot": 62737,
+    "building": 62738,
+    "store": 62739,
+    "album": 62740,
+    "palmtree": 62759,
+    "tickets-alt": 62756,
+    "money": 62758,
+    "thumbs-up": 62761,
+    "thumbs-down": 62786,
+    "layout": 62776,
+    "paperclip": 62790,
+    "email-alt2": 61706,
+    "menu-alt": 61707,
+    "plus-light": 61708,
+    "trash": 61709,
+    "uniF143": 61710,
+    "uniF494": 61711
```

Comment:
These should probably get the correct names (or removed, if they're duplicates):

![screen shot 2017-04-12 at 2 53 10 pm](https://cloud.githubusercontent.com/assets/541093/24974326/c5b006be-1f8f-11e7-8f04-4198291671c5.png)